models/AppState.py

```diff
@@ -1,5 +1,77 @@
 """Application state class"""
 
+import sys
+from numpy import array as np_array, min as np_min, ptp as np_ptp
+from models.PyCryptoBot import PyCryptoBot
+from models.TradingAccount import TradingAccount
+from models.exchange.binance import AuthAPI as BAuthAPI
+from models.exchange.coinbase_pro import AuthAPI as CAuthAPI
+
+def binanceMinimumOrderBase(app:PyCryptoBot, account:TradingAccount):
+    app = PyCryptoBot(exchange='binance')
+    api = BAuthAPI(app.getAPIKey(), app.getAPISecret(), app.getAPIURL())
+    info = api.client.get_symbol_info(symbol='BTCGBP')
```

Comment:
Is that correct to keeping symbol hardcoded?

Reply:
Whoops,no, Well spotted. Thanks